bcolz/toplevel.py

```diff
@@ -259,6 +259,21 @@ def fill(shape, dflt=None, dtype=np.float, **kwargs):
 
     """
 
+    def fill_helper(obj, dtype=None, length=None):
+        """Helper function to fill a carray with default values"""
+        assert issubclass(obj.__class__, bcolz.carray)
```

Comment:
None, I guess I was somehow subconsciously influenced by the topic subclassing carray https://github.com/Blosc/bcolz/issues/238